cancer_ai/utils/config.py

```diff
@@ -237,12 +237,6 @@ def add_common_args(cls, parser):
         default="./config/competition_config.json",
     )
 
-    parser.add_argument(
-        "--hf_repo_type",
```

Comment:
Im thinkin, if we are not using that one anymore we could probably also not pass it onto chain and hardcode 'model' type same way we have it in miner.py now

Reply:
yes, but it's for another change. miners would need to resubmit their models